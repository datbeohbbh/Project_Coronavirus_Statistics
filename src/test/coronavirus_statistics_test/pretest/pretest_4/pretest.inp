9 6
q 1 7 19
? 7
q 1 6 4
q 3 6 1
? 6
? 7