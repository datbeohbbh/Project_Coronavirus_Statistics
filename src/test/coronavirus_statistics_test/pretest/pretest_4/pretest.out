Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 1 people

